crates/tauri-plugin-holochain-service/android/src/main/java/com/plugin/holochain_service/HolochainServicePlugin.kt

```diff
@@ -56,11 +65,12 @@ class HolochainServicePlugin(private val activity: Activity): Plugin(activity) {
     @Command
     fun start(invoke: Invoke) {
         Log.d(TAG, "start")
-        this.serviceClient = HolochainServiceClient(
-            this.activity,
-            "com.plugin.holochain_service.HolochainService",
-            "org.holochain.androidserviceruntime.app"
-        )
+
+        // Start service
```

Comment:
Previously the client library would start the holochain service, now tauri-plugin-holochain-service starts the service.